netmanager/src/views/components/DataDisplay/DeviceView/DeviceLogs.js

```diff
@@ -568,6 +568,7 @@ export default function DeviceLogs({ deviceName, deviceLocation }) {
       <div>
         {show.logTable && (
           <MaintenanceLogsTable
+            isLoading={maintenanceLogs.length === 0}
```

Comment:
The assumption here is that we shall always have values maintenanceLogs. This could however bring issues if its empty. It would be even better to use the state from the device reducer (`netmanager/src/redux/DeviceManagement/reducers/devices.js`)

Reply:
Thanks for the insight. Lemme me fix that

Reply:
@Codebmk, a correction on my proposed approach above. The states to use are not currently in place but would depend on `LOAD_MAINTENANCE_LOGS_SUCCESS` (in netmanager/src/redux/DeviceRegistry/reducers/maintenanceLogs.js) for the maintenace logs table, and `LOAD_ALL_DEVICES_SUCCESS` (in netmanager/src/redux/DeviceRegistry/reducers/device.js) for the device registry table. You would also need to create necessary selectors for each. 
Seems like an over kill so you could use a simpler alternative for now.